projects/Canaille/default.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+
+{
+  metadata = {
+    summary = "Canaille is a zero-knowledge opinionated identity server.";
+    subgrants = [
+      "Canaille"
+    ];
+  };
+
+  nixos.modules.programs = {
+    Canaille = {
+      name = "Canaille";
+      module = ./programs/Canaille/module.nix;
+      examples.basic = {
+        module = ./programs/Canaille/examples/basic.nix;
+        description = "";
+        tests.basic = import ./programs/Canaille/tests/basic.nix args;
+      };
+      links = {
+        build = {
+          text = "Build from source";
+          url = "https://github.com/NixOS/nixpkgs/blob/1750f3c1c89488e2ffdd47cab9d05454dddfb734/pkgs/by-name/ca/canaille/package.nix#L134";
+        };
+        test = {
+          text = "Usage example";
+          url = "https://gitlab.com/yaal/canaille/#locally";
+        };
+      };
+    };
+
+    Canaille-cli = null;
+  };
+
+  nixos.modules.services = {
+    Canaille = {
+      name = "Canaille";
+      module = "${sources.inputs.nixpkgs}/nixos/modules/services/security/canaille.nix";
+      examples.basic = {
+        module = ./services/Canaille/examples/basic.nix;
+        description = "";
+        tests.basic = import ./services/Canaille/tests/basic.nix args;
```

We don't have to write a new test since we already have one in nixpkgs. Also delete the test file.

```suggestion
        tests.canaille = "${sources.inputs.nixpkgs}/nixos/tests/canaille.nix";
```

---

Resolved